### PR TITLE
add configurable homepage button to navbar

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Link zum Portal im Geo Engine Header ala GFBio in VAT](https://trello.com/c/wGl4MQnr/1546-link-zum-portal-im-geo-engine-header-ala-gfbio-in-vat)

File: projects/gis/src/app/main/main.component.html [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
 
+    <ng-container *ngIf="config.BRANDING.HOMEPAGE as homepage">
+        <div class="divider"></div>
+
+        <a [href]="homepage.URL" mat-flat-button color="primary" class="homepage-button">
+            <mat-icon>keyboard_return</mat-icon>
+            <span></span>
+            <img [src]="homepage.BUTTON_IMAGE_URL" [alt]="homepage.BUTTON_ALT_TEXT" [matTooltip]="homepage.BUTTON_TOOLTIP_TEXT" />
+        </a>
+    </ng-container>
```
Comment (ChristianBeilschmidt) [inline projects/gis/src/app/main/main.component.html:14]: If you want to be fresh, use

```suggestion
    @if (config.BRANDING.HOMEPAGE; as homepage) {
        <div class="divider"></div>

        <a [href]="homepage.URL" mat-flat-button color="primary" class="homepage-button">
            <mat-icon>keyboard_return</mat-icon>
            <span></span>
            <img [src]="homepage.BUTTON_IMAGE_URL" [alt]="homepage.BUTTON_ALT_TEXT" [matTooltip]="homepage.BUTTON_TOOLTIP_TEXT" />
        </a>
    }
```

File: projects/gis/src/app/main/main.component.scss [hunk 2]
```diff
@@ -31,6 +31,21 @@ mat-toolbar.navbar {
     }
 }
 
+.homepage-button {
+    z-index: 2;
+
```
Comment (ChristianBeilschmidt) [inline projects/gis/src/app/main/main.component.scss:36]:
```suggestion
```

I guess this is not necessary?

File: projects/gis/src/app/main/main.component.html [hunk 3]
```diff
@@ -3,6 +3,16 @@
 
     <geoengine-layer-list-menu></geoengine-layer-list-menu>
 
+    <ng-container *ngIf="config.BRANDING.HOMEPAGE as homepage">
+        <div class="divider"></div>
+
+        <a [href]="homepage.URL" mat-flat-button color="primary" class="homepage-button">
+            <mat-icon>keyboard_return</mat-icon>
+            <span></span>
+            <img [src]="homepage.BUTTON_IMAGE_URL" [alt]="homepage.BUTTON_ALT_TEXT" [matTooltip]="homepage.BUTTON_TOOLTIP_TEXT" />
```
Comment (ChristianBeilschmidt) [inline projects/gis/src/app/main/main.component.html:12]: In fact, you can use it as text only, but you have two types of font colors

grafik

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/gis/src/app/main/main.component.html (hunk 3 above)
Comment (michaelmattig) [reply projects/gis/src/app/main/main.component.html:12]: efa38afd738ffc393da016aa8adf32354cd71291

**Comment by ChristianBeilschmidt** [comment]
Super strange regarding the style bundle size.
I hope this helps

[git diff](https://github.com/geo-engine/geoengine-ui/files/13717660/chris.txt)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**